dr_trigger.py

```diff
@@ -42,7 +71,9 @@ def trigger_dump(s_pid, s_port, path):
         sys.exit(1)
 
     sock = connect_to_server(server_pid)
-    request_dump(sock, port, file.fileno())
+    if sock:
+        request_dump(sock, port, file.fileno())
+
     file.close() 
     sock.close()
```

Comment:
What happens if connect_to_server() fails?
sock is none, but what will sock.close() do?